api/events.py

```diff
@@ -211,9 +211,11 @@ def post(self):
                     if run.status == "created":
                         run.create_benchmark_build()
 
-                notification = benchmarkable.pull_notification()
-                notification.create_pull_comment(
-                    notification.generate_pull_comment_body()
+                Github(repo).create_pull_comment(
+                    pull_number,
+                    f"Benchmark runs are scheduled for commit {benchmarkable.id}. Watch "
+                    f"{Config.CONBENCH_URL} for updates. A comment will be posted here "
+                    "when the runs are complete.",
                 )
```

Comment:
This is where ursabot immediately responds to someone who wrote `@ursabot please run benchmarks`